Learn-CSS-Blog/index35.html

```diff
@@ -0,0 +1,80 @@
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed imperdiet 
+                        fermentum tortor, at aliquet ante varius nec. Sed vehicula libero sed 
+                        ligula varius, sit amet eleifend velit tincidunt. Fusce eu justo quis 
+                        neque sagittis venenatis. Vivamus ac elit ultricies, laoreet leo vitae, 
+                        rutrum eros. Mauris malesuada auctor quam et facilisis. Nullam ipsum massa,
+                        ultricies id faucibus non, dapibus eu arcu. Fusce non interdum risus.
+                        Ut at lacus mattis, aliquam tellus at, luctus magna. Quisque euismod, 
+                        eros non dictum tincidunt, nisl mauris elementum nibh, vitae facilisis 
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed imperdiet 
+                        fermentum tortor, at aliquet ante varius nec. Sed vehicula libero sed 
+                        ligula varius, sit amet eleifend velit tincidunt. Fusce eu justo quis 
+                        neque sagittis venenatis. Vivamus ac elit ultricies, laoreet leo vitae, 
+                        rutrum eros. Mauris malesuada auctor quam et facilisis. Nullam ipsum massa,
+                        ultricies id faucibus non, dapibus eu arcu. Fusce non interdum risus.
+                        Ut at lacus mattis, aliquam tellus at, luctus magna. Quisque euismod, 
+                        eros non dictum tincidunt, nisl mauris elementum nibh, vitae facilisis 
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                </div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Nested a `<link>` tag below the `</title>` tag. Add a `rel` attribute and give
```

Comment:
Add a `<link>` tag below the closing `</title>` tag. Add a `rel` attribute and give
    it the value "stylesheet". The `rel` attribute specifies the relationship between
    the current document and the linked document.